corehq/apps/locations/adjacencylist.py

```diff
@@ -182,56 +180,26 @@ def root_nodes(self):
         parent_col = self.model.parent_id_attr
         return self.all().filter(**{parent_col + "__isnull": True})
 
-    def _mptt_get_queryset_ancestors(self, node, *args, **kw):
-        if isinstance(node, Q):
-            node = self.filter(node)
-        return super(AdjListManager, self).get_queryset_ancestors(node, *args, **kw)
-
-    def _mptt_get_queryset_descendants(self, node, *args, **kw):
-        if isinstance(node, Q):
-            node = self.filter(node)
-        return super(AdjListManager, self).get_queryset_descendants(node, *args, **kw)
-
     def get_queryset_ancestors(self, queryset, include_self=False):
         timing = TimingContext("get_queryset_ancestors")
-        mptt_qs = cte_qs = queryset
+        cte_qs = queryset
         if isinstance(queryset, ComparedQuerySet):
-            mptt_qs = queryset._mptt_set
             cte_qs = queryset._cte_set
-        if settings.IS_LOCATION_CTE_ONLY:
-            mptt_set = None
-        else:
-            with timing("mptt"):
-                mptt_set = self._mptt_get_queryset_ancestors(mptt_qs, include_self)
-        if settings.IS_LOCATION_CTE_ENABLED:
-            with timing("cte"):
-                cte_set = self._cte_get_queryset_ancestors(cte_qs, include_self)
-        else:
-            cte_set = None
+        mptt_set = None
+        with timing("cte"):
```

Comment:
are we eventually planning on removing timing?

Reply:
yes, once I've verified that this PR does not affect timing (I don't expect it will, but just being safe)